### ISSUE TITLE
Handle "Branding" between PyPI and Test PyPI

### ISSUE THREAD

**Issue by dstufft** [pypa/warehouse#834]
Right now we have the site name configuration, however that isn't going to work with ``L20n.js``. Probably we should figure out something like ``/locales/branding.{{ locale }}.l20n`` which will just have something like:

```html
<brandName {
    *short: "PyPI"
    long: "Python Package Index"
}>
```

or

```html
<brandName {
    *short: "TestPyPI"
    long: "Test Python Package Index"
}>
```

We'll need to figure out how this will work and what the right way of doing it will be. This will also be affected by #833 since if we are going to support default translations in the HTML, then the templates will need access to the site name (both short and long) as well.

**Comment by dstufft** [comment]
We're not going to launch with ``L20n.js`` so this isn't a critical launch task.

**Comment by dstufft** [comment]
This was supposed to be closed with #1335.
